### ISSUE TITLE
Cmd+C from URL broken with keyboard

### ISSUE THREAD

**Issue by vpetersson** [brave/brave-ios#1146]
### Description: 

This is a minor (but annoying) bug that I encounter several times per week and would love to see it resolved. 

Tl;dr:  It’s not possible to copy the URL from the address bar using the keyboard (on iPad).

### Steps to Reproduce 
  1. Navigate to any website
  2. Click at the address bar and press cmd+a to select the URL (this part works fine)
  3. Press cmd+c on the keyboard to copy the URL 

**Actual result:** 

Nothing is populated into the clipboard.

**Expected result:**

The address to be copied.

**Reproduces how often:** 

Easily reproduced

**Brave Version:** 

Version 1.9.3 (19.05.22.15)

**Device details:** 

iPad Pro (10.5-inch) with Apple’s Smart Keyboard

(Previous generation)

**Website problems only:**

Unrelated to website.

### Additional Information 

As a test, I tried to reproduce this on Safari to rule out that it’s a keyboard issue, but was unable to do so. There the copy’ing works just find.

**Comment by andrewcking** [comment]
I am also seeing this issue - would love to see it resolved. For anyone on iPad this is a very common occurrence.

**Comment by kylehickinson** [comment]
This has actually been fixed as a side-effect of #1990 merging :)

**Comment by kjozwiak** [comment]
Verification **PASSED** on `iPhone 6s` running `iOS 13.2.2` using `1.14 (19.11.20.19)`

* ensured that `CMD` + `T` opened a new tab without issues
* ensured that `CMD` + `A` selected the entire URL without any issues
* ensured that `CMD` + `C` copied the URL without any issues
* ensured that `CMD` + `X` cut the URL without any issues
* ensured that `CMD` + `V` pasted the URL from the clipboard without any issues

Verification **PASSED** on `iPhone 6s+` running `iOS 12.4.1` using `1.14 (19.11.20.19)`

* ensured that `CMD` + `T` opened a new tab without issues
* ensured that `CMD` + `A` selected the entire URL without any issues
* ensured that `CMD` + `C` copied the URL without any issues
* ensured that `CMD` + `X` cut the URL without any issues
* ensured that `CMD` + `V` pasted the URL from the clipboard without any issues

Verification **PASSED** on `iPad Air 3rd Gen` running `iOS 13.3` using `1.14 (19.11.20.19)`

* ensured that `CMD` + `T` opened a new tab without issues
* ensured that `CMD` + `A` selected the entire URL without any issues
* ensured that `CMD` + `C` copied the URL without any issues
* ensured that `CMD` + `X` cut the URL without any issues
* ensured that `CMD` + `V` pasted the URL from the clipboard without any issues

Verification **PASSED** on `iPad Mini 4` running `iOS 12.4.1` using `1.14 (19.11.20.19)`

* ensured that `CMD` + `T` opened a new tab without issues
* ensured that `CMD` + `A` selected the entire URL without any issues
* ensured that `CMD` + `C` copied the URL without any issues
* ensured that `CMD` + `X` cut the URL without any issues
* ensured that `CMD` + `V` pasted the URL from the clipboard without any issues

**Comment by srirambv** [comment]
Verification passed on iPhone XR with iOS 13.2 running 1.14(19.11.26.13)
* Verified <kbd>CMD</kbd>+<kbd>A</kbd> selects the entire URL without any issues
* Verified <kbd>CMD</kbd>+<kbd>T</kbd> opens a new tab
* Verified <kbd>CMD</kbd>+<kbd>C</kbd> copies the URL without issues
* Verified <kbd>CMD</kbd>+<kbd>X</kbd> cut the URL without issues
* Verified <kbd>CMD</kbd>+<kbd>L</kbd> sets the focus on URL bar when a page is loaded
* Verified <kbd>CMD</kbd>+<kbd>V</kbd> pastes text from clipboard without issue
